src/site/content/en/progressive-web-apps/progressively-enhance-your-pwa/index.md

```diff
@@ -14,7 +15,6 @@ scheduled: true
 date: 2020-06-29
 # updated: 2020-06-15
 tags:
-  - blog
```

Comment:
Let's keep the `blog` tag so that it shows up in both places

Reply:
Ah, I thought it was either or. Cool!